### PR TITLE
Allow passing named arguments to custom commands (alias).

### PR DESCRIPTION
**Pull request by jkulhanek**
This allows passing named arguments to custom commands. An example would be `pubs search keyword --ignore-author` and the corresponding alias would redirect the call to the custom script like follows: `search = !"$DOTFILES/pubs/scripts/search.py" "$@"`.

### CODE REVIEW

**Review by omangin — COMMENTED** [review]
Thanks @jkulhanek  for fixing this! and sorry for the late review.

Do you think you could add a quick unit test to https://github.com/pubs/pubs/blob/master/tests/test_plug_alias.py to prevent regressions?

**Comment by jkulhanek** [comment]
A simple test added.

File: tests/test_plug_alias.py [hunk 1]
```diff
@@ -61,6 +61,19 @@ def testShellAliasEscapes(self):
             shlex.split(self.subprocess.called.splitlines()[-1])[1:],
             args)
 
+    def testShellAliasNamedArguments(self):
+        parser = argparse.ArgumentParser()
+        parser.add_argument('--test2')
```
Comment (omangin) [inline tests/test_plug_alias.py:66]: Did you intend to use this one?

**Review by omangin — APPROVED** [review]
See minor comments. Thanks @jkulhanek  for adding the test!

**Comment by benureau** [comment]
Rebased an pushed to master as 2e939dbc and 0777a99d30. Apologies for the long delay.

**Pull request closed without merging**